base/document-download-frontend-deployment.yaml

```diff
@@ -54,7 +54,24 @@ spec:
               value: '$(NEW_RELIC_MONITOR_MODE)'
             - name: NOTIFY_ENVIRONMENT
               value: '$(ENVIRONMENT)'
-          resources: {}
+          resources:
+            requests:
+              cpu: "50m"
+              memory: "400Mi"
+            limits:
+              cpu: "400m"
+              memory: "800Mi"
+          ports:
+            - containerPort: 7001
+          readinessProbe:
+            httpGet:
+              path: /_status
```

Comment:
Does not call other services https://github.com/cds-snc/notification-document-download-frontend/blob/3a650d03d3f64c789d3bcebcb66624ceb0555523/app/main/views/index.py#L11-L13